### PR TITLE
Add beginners course link from KodeKloud

### PR DESCRIPTION
**Pull request by mmumshad**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
@mmumshad Thanks for the update.
I checked the changes on https://deploy-preview-12190--kubernetes-io-master-staging.netlify.com/docs/tutorials/online-training/
/lgtm

File: content/en/docs/tutorials/online-training/overview.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ Here are some of the sites that offer online training for Kubernetes:
 
 * [Kubernetes the Hard Way (LinuxAcademy.com)](https://linuxacademy.com/linux/training/course/name/kubernetes-the-hard-way)
 
-* [Certified Kubernetes Application Developer Preparation Course (KodeKloud.com)](https://kodekloud.com/p/kubernetes-certification-course)
+* [Kubernetes for the Absolute Beginners with Hands-on Labs (KodeKloud.com)](https://kodekloud.com/p/kubernetes-for-the-absolute-beginners-hands-on?coupon_code=KUBEIO)
```
Comment (jimangel) [inline content/en/docs/tutorials/online-training/overview.md]: What is the purpose of the coupon_code?

Comment (mmumshad) [reply content/en/docs/tutorials/online-training/overview.md]: It gives a special discount to students while enrolling in the course.  Can be removed if not appropriate.

**Comment by Rajakavitha1** [comment]
I checked : https://deploy-preview-12190--kubernetes-io-master-staging.netlify.com/docs/tutorials/online-training/
/lgtm
/assign @jimangel  please take a look

**Pull request merged**